src/main/kotlin/no/nav/familie/ef/iverksett/lagretilstand/LagreTilstandServiceJdbc.kt

```diff
@@ -0,0 +1,92 @@
+package no.nav.familie.ef.iverksett.lagretilstand
+
+import no.nav.familie.ef.iverksett.domene.TilkjentYtelse
+import no.nav.familie.kontrakter.felles.objectMapper
+import org.slf4j.LoggerFactory
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate
+import org.springframework.stereotype.Repository
+import java.util.*
+
+@Repository
+class LagreTilstandServiceJdbc(val namedParameterJdbcTemplate: NamedParameterJdbcTemplate) : LagreTilstand {
+
+    private val secureLogger = LoggerFactory.getLogger("secureLogger")
+
+    override fun lagreTilkjentYtelseForUtbetaling(behandlingId: UUID, tilkjentYtelseForUtbetaling: TilkjentYtelse) {
+        val sql = "insert into iverksett_resultat values(:behandlingId, null, :tilkjentYtelseForUtbetaling::json, null)"
+
+        val tilkjentYtelseForUtbetalingJson = objectMapper.writeValueAsString(tilkjentYtelseForUtbetaling)
+        val mapSqlParameterSource = MapSqlParameterSource(
```

Comment:
Her er det også mulig å bruke denne, er smaksak :) 
```
val mapSqlParameterSource = MapSqlParameterSource()
                .addValue("behandlingId", behandlingId)
                .addValue("tilkjentYtelseForUtbetaling", tilkjentYtelseForUtbetalingJson)
```

Reply:
Ja, jeg endrer til denne. Ser litt bedre ut, i alle fall når det er bare to parametere